transmart_loader/copy_writer.py

```diff
@@ -285,6 +314,13 @@ def visit_visit(self, visit: Visit) -> None:
             self.encounter_mappings_writer.writerows(encounter_mapping_rows)
             self.visits[visit.identifier] = encounter_num
 
+    def write_tree_node_tags(self, metadata: TreeNodeMetadata, node_path: str):
+        for tag_type, tag in metadata.values.items():
```

Comment:
For all other entities we first check if the item has already been written, e.g., by
```python
tag_key = TagKey(node_path, tag_type)
if tag_key not in self.tags:
  # write tag, add tag_key to self.tags
```